### ISSUE TITLE
Travel Form: error navigating to "Confirmation" wizard page

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#6681]
Error in log:

```
Error tag name "xxf:blur" is not allowed. Possible tag names are: <help>,<load>,<submission>(schema: oxf:/ops/xforms/x|
|forms-server-response.rng)
```

**Comment by ebruchez** [comment]
Works with 2023.1.5.

**Comment by ebruchez** [comment]
Regressed with bad cherry-pick for my fix for #6679.
